com.google.AndroidStudio.json

```diff
@@ -53,12 +53,12 @@
                 {
                     "type": "extra-data",
                     "filename": "android-studio.tar.gz",
-                    "size": 907569312,
+                    "size": 907537313,
                     "only-arches": [
                         "x86_64"
                     ],
-                    "url": "https://dl.google.com/dl/android/studio/ide-zips/4.0.1.0/android-studio-ide-193.6626763-linux.tar.gz",
-                    "sha256": "f2f82744e735eae43fa018a77254c398a3bab5371f09973a37483014b73b7597"
+                    "url": "https://redirector.gvt1.com/edgedl/android/studio/ide-zips/4.0.2.0/android-studio-ide-193.6821437-linux.tar.gz",
```

Comment:
@burnermanx Gracias por el aporte!
Could you please change the url to dl.google.com?

Reply:
Done